content/master/guides/implementing-safe-start.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Implementing safe-start in Providers  
+weight: 90
+state: alpha
+alphaVersion: 2.0
+description: Guide for provider developers to add safe-start capability for
+  selective resource activation
+---
+
+This guide shows provider developers how to implement safe-start capability in
+their Crossplane providers. safe-start enables
+[disabling unused managed resources]({{<ref "disabling-unused-managed-resources">}})
+through ManagedResourceDefinitions, improving performance and reducing resource
+overhead.
+
+{{<hint "important">}}
+safe-start requires Crossplane v2.0+ and crossplane-runtime v2.0+.
+Implementing safe-start involves code changes that affect provider startup
+behavior.
+{{</hint>}}
+
+## What safe-start provides
+
+safe-start changes how your provider handles CRD installation:
```

Comment:
```suggestion
Safe-start changes how your provider handles CRD installation:
```

or `safe-start`? Not sure, but I didn't add suggestions for the other instances.